recipe/build.sh

```diff
@@ -11,7 +11,7 @@ then
     export LINKFLAGS="${LINKFLAGS} -stdlib=libc++ -std=c++11"
 fi
 
-./configure --prefix="${PREFIX}"
+./configure --prefix="${PREFIX}" --enable-unicode
```

Comment:
Do you know which one you need? We could add pcre2. I think there is a question of whether we add it to this feedstock as a separate branch or a totally new feedstock.